sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java

```diff
@@ -184,7 +190,9 @@ private BeamRelNode convertToBeamRelInternal(String sql, QueryParameters queryPa
     RelMetadataQuery.THREAD_PROVIDERS.set(
         JaninoRelMetadataProvider.of(root.rel.getCluster().getMetadataProvider()));
     root.rel.getCluster().invalidateMetadataQuery();
-    return (BeamRelNode) plannerImpl.transform(0, desiredTraits, root.rel);
+    BeamRelNode beamRelNode = (BeamRelNode) plannerImpl.transform(0, desiredTraits, root.rel);
+    LOG.info("BEAMPlan>\n" + RelOptUtil.toString(beamRelNode));
```

Comment:
Remove this LOG.info before merge the code?

Reply:
I saw we do the same in the Calcite planner: https://github.com/apache/beam/blob/a443fdd4c2bd48c666478168cd1edb4db661103d/sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/CalciteQueryPlanner.java#L190

Should we downgrade both to the `debug` level?

Reply:
I see. Maybe just leave this line here. I don't know where in the past we don't have this logging but match with what CalcitePlanner does is ok.